src/reachy2_sdk/media/camera.py

```diff
@@ -55,8 +86,15 @@ def get_parameters(
     ) -> Optional[
         Tuple[int, int, str, npt.NDArray[np.float64], npt.NDArray[np.float64], npt.NDArray[np.float64], npt.NDArray[np.float64]]
     ]:
-        """Returns the height, width, distortion model, distortion coefficients, instrinsic matrix,"""
-        """ rotation matrix and projection matrix"""
+        """Retrieve camera parameters including intrinsic and extrinsic matrices.
```

Comment:
this function doesn't return the extrinsic matrix, it's the function below, am I wrong ?